docs/user/dql/basics.rst

```diff
@@ -363,6 +363,17 @@ SQL query::
 	{
 	  "query" : "SELECT account_number FROM accounts/account"
 	}
+Example 4: Selecting From Index using Partition Shard
```

Comment:
What happens when `PARTITION` is provided for another data source, like Prometheus or Spark?

Reply:
They will need to implement a sharing or partition option separately.  Most datasources would have such an option.